src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer machine started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+
+        System.out.println("Bulldozer machines stopped working");

```

Comment:
The message in 'System.out.println' should be more informative and indicate the type of machine. Consider specifying that it is a Bulldozer that has stopped working. Also, there's a typo in the word 'machines', it should be singular 'machine'.